### DISCUSSION TITLE
诶嘿

### DISCUSSION THREAD

**Discussion by HeroBrine-c** [share121/inter-knot#2209] [其他]
![微信图片_20240815154430](https://github.com/user-attachments/assets/09ed6a4a-fe2d-497b-a24a-6e400076823e)

https://github.com/user-attachments/assets/06419a52-bfbe-4d23-82db-d8cfbbc1f622

# **无不良引导！！**

**Comment by BSNHD** [comment]
其实你完全可以右键然后新标签页打开图片的。你演示的这个应该算是bug
